src/common/components/hosts/types.ts

```diff
@@ -9,7 +9,7 @@ export type ClusterHostsTableProps = {
   setDiscoveryHintModalOpen?: HostsNotShowingLinkProps['setDiscoveryHintModalOpen'];
 };
 
-export type HostUpdateParams = {
+export type InternalHostUpdateParams = {
```

Comment:
Looking at the usage of this type, I'd favor naming it `EditHostFormValues`
```suggestion
export type EditHostFormValues = {
```